lib/utilities/submitting_button_icon.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter/material.dart';
+
+class SubmittingButtonIcon extends StatelessWidget {
```

Comment:
https://github.com/itchysats/10101/pull/398 is introducing an AsyncButton, which we should use eventually.